### DISCUSSION TITLE
connect ECONNREFUSED 127.0.0.1:5000

### DISCUSSION THREAD

**Discussion by hardik92** [DevSkillsHQ/discusssions#49] [Help] [answered]
getting this error in actions pipeline

connect ECONNREFUSED 127.0.0.1:5000

----update:

I checked the logs and the problem is with the python version

My code is compatible with Python 3.9.18.

Please suggest on how to update the setup python action.

**Comment by fenske** [comment, marked as answer]
Hi @hardik92. I've downgraded the Python version on your branch to match the one you use. However, the same issue remains (though, it doesn't seem to be related to the environment). See the PR for more details.

**Reply by hardik92** [reply]
Thanks for the help @fenske. I downgraded itsdangerous and it worked.
Now react is failing to connect.
connect ECONNREFUSED 127.0.0.1:5173

**Reply by fenske** [reply]
Try running the app using GitHub Codespaces as per instructions. 

When I did that, it failed to render:

![CleanShot 2023-11-08 at 15 03 08@2x](https://github.com/DevSkillsHQ/discusssions/assets/1162212/6ff65d3a-8161-4961-a1e4-c0253d5255db)
